src/core/globals.hpp

```diff
@@ -21,7 +21,8 @@ namespace otto {
       none = 0,
       ui_closed,
       user_exit,
-      audio_error
+      audio_error,
+      graphics_error
```

Comment:
Yeah, that's another thing that probably needs a good scrubbing -- there are a handful of things that can cause the app to exit, and there are multiple ways of the app exiting currently.  It would be good to standardize all the shutdown procedures.